core/domain/user_jobs_one_off_test.py

```diff
@@ -2073,3 +2073,91 @@ def test_job_removes_deleted_exp_from_edited_explorations(self):
         self.assertEqual(model_instance_2.created_exploration_ids, [])
         self.assertEqual(
             model_instance_2.edited_exploration_ids, [])
+
+
+class VerifyProfilePictureOneOffJobTests(test_utils.GenericTestBase):
+
+    PNG_IMAGE_WRONG_DIMENSIONS_BASE64 = (
```

Comment:
These constants are quite long. Could we maybe store them as files & load them, instead? It makes the tests harder to read, and they're not super "interesting" from a testing perspective (e.g. humans can't really look at these and verify their correctness--we just have to assume that they're set up correctly for the purpose of the tests).